controllers/component_build_controller_pac.go

```diff
@@ -501,12 +512,14 @@ func (r *ComponentBuildReconciler) ConfigureRepositoryForPaC(ctx context.Context
 			// Webhook
 			ghclient = github.NewGithubClient(accessToken)
 
-			log.Info("Setup Pipelines as Code webhook for")
 			err = github.SetupPaCWebhook(ghclient, webhookTargetUrl, webhookSecret, owner, repository)
 			if err != nil {
+				log.Error(err, "failed to setup PaC webhook", l.Audit, "true")
```

Comment:
would it make sense to add an info about the related webhook url and component/namespace for easier debugging?

Reply:
Thank you, good idea. I will add information about webhook. Information about component/namespace is already included into context.